moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1178,9 +1132,8 @@ as the new values that correspond to the group */
      @param jump_threshold The struct holding jump thresholds to determine if a joint space jump has occurred.
```

Comment:
for all the old cartesian-related functions, add comments saying as of ROS-Melodic they are deprecated and should not be used